diff --git a/lib/Perl/Dist/Strawberry/Step.pm b/lib/Perl/Dist/Strawberry/Step.pm
index 2c39bda..98f25bc 100644
--- a/lib/Perl/Dist/Strawberry/Step.pm
+++ b/lib/Perl/Dist/Strawberry/Step.pm
@@ -610,9 +610,8 @@ sub _update_config_gc {
         elsif ($line =~ /^PERL/) {
             push @perl_lines, $line;
         }
-        else {
-            $line =~ m/^([\w]+)=(.+)$/;
-            $data{$1} = $2;
+        elsif ($line =~ m/^([\w]+)=(.*)$/) {
+            $data{$1} = length($2) ? $2 : "''";
         }
     }
 
